### ISSUE TITLE
Bug when used with flexslider

### ISSUE THREAD

**Issue by Chryogenic** [FrDH/jQuery.dotdotdot#132]
I'm trying to use this with a JS flexslider that auto plays through various images and text.
To change slides there is a timer that simply changes the z-index and opacity of the next image.

The library works fine, however if the slide changes while dotdotdot is watching and the window is resized, I get the following error every second in the console:
> Uncaught TypeError: Cannot read property 'width' of undefined at t.(anonymous function)._watchSizes.

I was using the following

**JS**
```
$(document).ready(function () {

        $(".test").dotdotdot({

            ellipsis: "\u2026 ",
          
            truncate: "word",
          
            keep: null,
          
            watch: "true",
          
            tolerance: 40
          });
    });
```

**Comment by FrDH** [comment]
My guess is, the plugin looses the reference to the ".test" node when the slider moves the slide in the HTML. Is there an online example I could have a look at?

**Comment by rlenders** [comment]
That error is the same error that was fixed with #134, so this is probably fixed too.

**Comment by FrDH** [comment]
Could you please check if the bug is fixed the latest version?
